src/js/components/Carousel/doc.js

```diff
@@ -20,16 +20,21 @@ export const doc = Carousel => {
 
   DocumentedCarousel.propTypes = {
     ...genericProps,
+    activeIndex: PropTypes.number.description(
```

Comment:
Can we rename this to `activeChild`, to align with `initialChild` and `onChild`? In retrospect, it would have been better to use "Index" as the suffix instead of "Child". But, here we are.